picard/ui/infodialog.py

```diff
@@ -286,12 +286,10 @@ def format_file_info(file_):
     info.append((_("Filename:"), file_.filename))
     if '~format' in file_.orig_metadata:
         info.append((_("Format:"), file_.orig_metadata['~format']))
-    try:
-        size = os.path.getsize(encode_filename(file_.filename))
+    if '~filesize' in file_.orig_metadata:
+        size = file_.orig_metadata['~filesize']
```

Comment:
Since you removed the call to `os.path.getsize()` importing `os.path` at top of the file isn't needed anymore.